src/pages/auth/reset-password.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from "react";
+import { useNavigate, useParams } from "react-router-dom";
+import { useAuth } from "../../data/authentication";
+import { createToastify } from "../../helpers/toastify";
+
+export const ResetPassword = () => {
+  const params = useParams();
+  const [password, setPassword] = useState({ new: "", confirm: "" });
+  const navigate = useNavigate();
+  const { resetPassword } = useAuth();
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setPassword({
+      ...password,
+      [event.target.name]: event.target.value,
+    });
+  };
+
```

Comment:
Let's add some logic to redirecto to login if token or userId is not in the queryParams